plugins/vendor/zoom/lib/api/webPage/KalturaZoomRegistrationPage.html

```diff
@@ -161,96 +74,190 @@
             });
         }
         function  onSuccess() {
-            $('#submitError').hide();
-            $('#submitOK').show();
+            alert("Saved Successfully");
         }
         function onFail(errorMessage) {
-            $('#submitOK').hide();
-            $('#submitError').show();
-            $('#submitError').html(errorMessage);
+            alert("Data was Not Saved.\n"+errorMessage);
+        }
+        function onLoad()
+        {
+            console.log(partnerId);
+            if(!partnerId)
+            {
+                return;
+            }
+            setDefaultValues();
+            $.ajax({
+                url: getUrl,
+                type: 'post',
+                data: {"ks": ks, 'accountId': accountId ,'partnerId':partnerId },
+                success: function (res) {
+                    console.log(res);
+                    if (res) {
+                        if (res.objectType == 'KalturaAPIException') {
+                            alert("Error loading page.\n"+res.message);
+                        } else {
+                            //   onSuccess();
+                            console.log(res);
+                            loadParameters(res);
+                        }
+                    } else {
+                        //onSuccess();
+                    }
+                },
+                error: function () {
+                    alert("Error loading page.\n");
+                    console.log('err during api call');
+                }
+            });
         }
 
-    </script>
-    <script>
-        (function() {
-            document.addEventListener('DOMContentLoaded', function () {
-                var submitButton = document.querySelector('.submit-button');
-                submitButton.addEventListener('click', trySubmit);
+        function setDefaultValues(){
+            $('#enableWebinarUploads').prop('checked',true);
+            $('#enableRecordingUpload').prop('checked',true);
+            $('input:radio[name=zoomUserMatchingMode]').val(['0']).prop('checked',true);
+            $('input:radio[name=handleParticipantsMode]').val(['2']).prop('checked',true);
+        }
+        function loadParameters(p) {
+         jQuery.each(p, function(i,v) {
+                console.log(i,v);
+                switch (i) {
+                    case 'defaultUserId':
+                    case 'zoomCategory':
+                    case 'zoomUserPostfix':
+                    case 'zoomWebinarCategory':
+                        $('#'+i).val(v);
+                        break;
+                    case 'enableWebinarUploads':
+                    case 'enableRecordingUpload':
+                        $('#'+i).prop('checked', v);
+                        break;
+                    case 'zoomUserMatchingMode':
+                    {
+                        //check user matching mode
+                        if(v==3)
```

Comment:
i would use some const for this 3 here and on line 32 to make it more clear what is this mode